.github/workflows/ci.yml

```diff
@@ -6,9 +6,55 @@ on:
       - '*'
 
 jobs:
-  staticcheck:
+  build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        command: [compile-linux, compile-windows, compile-darwin]
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
```

Comment:
Just curious (here and below): Any reason not to use the latest go-version (1.15)?

Reply:
We use CodeBuild in our CD pipeline to build the binaries, which for now doesn't support go1.14 (https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-available.html).

To keep it consistent with the pipeline I've set it to 1.14 when they support 1.15 we can upgrade it :)